mandatory/2-function-creation.js

```diff
@@ -6,7 +6,15 @@ Complete the function to check if the variable `num` satisfies the following req
 Tip: use logical operators
 */
 
-function validate(num) {}
+function validate(num) {
+  if(typeof(num)!=='string' && num%2==0 && num<=100){
```

Comment:
`typeof(num) == 'number'` may be better to account for other possibilities (e.g `false` would meet all the conditions here), but otherwise :+1: